test/unit/Distributor/executeCoverAction.js

```diff
@@ -0,0 +1,235 @@
+const { accounts, web3, artifacts } = require('hardhat');
+const { ether, expectEvent, expectRevert, time } = require('@openzeppelin/test-helpers');
+const { assert } = require('chai');
+const { toBN } = web3.utils;
+const { hex } = require('../utils').helpers;
+const { DEFAULT_FEE_PERCENTAGE } = require('./helpers');
+const { ZERO_ADDRESS } = require('@openzeppelin/test-helpers').constants;
+const { Assets: { ETH } } = require('../utils').constants;
+const BN = web3.utils.BN;
```

Comment:
Regroup this imports please. First import from packages, then from local files, afterwards define variables from imports (like `BN`, `toBN`). PS: `BN` doesn't seem to be used at all.